distribution/scenes/textures/patterns/agate.pov

```diff
@@ -2,35 +2,38 @@
 // To view a copy of this license, visit http://creativecommons.org/licenses/by/3.0/
 // or send a letter to Creative Commons, 444 Castro Street, Suite 900, Mountain View,
 // California, 94041, USA.
-
-// Persistence Of Vision raytracer sample file.
-// Agate pattern example
 //
-// -w320 -h240
+// Persistence Of Vision Ray Tracer ('POV-Ray') sample file.
+//
+// Agate pattern example agate.pov.
+//
 // -w800 -h600 +a0.3
 
 #version 3.7;
-
 global_settings { assumed_gamma 1.0 }
+#default { finish { ambient 0.006 diffuse 0.456 } }
 
-#include "colors.inc"
+#declare Black = srgb <0,0,0>;
+#declare White = srgb <1,1,1>;
 
-#declare T1=
- texture{
-   pigment{
-     agate color_map{[0.0 Black][1.0 White]}
-     scale 0.5
-   }
- }
+#declare T1 = texture {
+    pigment {
+        agate
+        color_map {
+            [0.0 Black]
+            [1.0 White]
+        }
+        scale 0.5
+    }
+}
```

Comment:
I understand and fully go along with your motivation. But as I said, if we go for unification of the style, I think we should first discuss what to actually go for. Not the least because such a unification would also touch areas beyond sample scenes, such as documentation (embedded samples) and even the POV-Ray source code (e.g. editor default settings like tab width, which I think I dimly recall as having the ghastly value of 3 in a fresh install).